backend/src/test/java/de/sambam/gamenighttracker/controller/UserControllerTest.java

```diff
@@ -66,15 +102,102 @@ public void getAllGamesFromDbTest() {
 
         //THEN
         assertEquals(getResponse.getStatusCode(), HttpStatus.OK);
-        assertThat(gameList.length, is(2));
+        assertThat(gameList.length, is(3));
         assertThat(List.of(gameList), containsInAnyOrder(
                 Game.builder()
                         .name("Bonfire")
                         .releaseYear("2020")
+                        .gameSessionList(List.of(GameSession.builder()
+                                .id("1")
+                                .sessionState("DONE")
+                                .winnerPlayerId("Mario")
+                                .playerList(List.of(player1, player2))
+                                .build()))
                         .build(),
                 Game.builder()
                         .name("Calico")
                         .releaseYear("2020")
+                        .gameSessionList(List.of(GameSession.builder()
+                                .id("2")
+                                .sessionState("DONE")
+                                .winnerPlayerId("samBAM")
+                                .playerList(List.of(player3, player4))
+                                .build()))
+                        .build(),
+                Game.builder()
+                        .name("Calico")
+                        .releaseYear("2020")
+                        .gameSessionList(List.of(GameSession.builder()
```

Comment:
if you use these values multiple times it might make sense to extract them into a private helper method which you use throughout your tests

Reply:
just noticed why it did not work at first, thanks ^^